docs/pages/config-reference.mdx

```diff
@@ -304,6 +304,12 @@ auth_service:
     license_file: /var/lib/teleport/license.pem
 
 
+    # Configures a banner message to be displayed to a user logging into the cluster, 
+    # which must be acknowledged before the user is allowed to log in. Note that will
+    # be shown *before* login, so should not contain any confidential information. 
+    # Defaults to the empty string, implying no message or acknowledgment is required.
+    message_of_the_day: ""
```

Comment:
This MOTD article for PAM might be worth reviewing/updating here: https://goteleport.com/docs/server-access/guides/ssh-pam/#set-message-of-the-day-motd-with-teleport. Looks like your changes are specific to the API?

Reply:
The message added by this change is shown as part of `tsh login` rather than on access to a resource, while it's a similar feature it happens at a different time. Will see if I can add anything useful to the PAM guide to refer this, though.

Reply:
I've added a note that you can add a cluster-wide MotD to the PAM documentation. I'm not certain that it's the 100% right place for it (as it doesn't use PAM at all) but at least someone searching for `motd` will find it :-)